src/sentry/integrations/utils/code_mapping.py

```diff
@@ -77,13 +76,20 @@ def filter_source_code_files(files: List[str]) -> List[str]:
 # XXX: Look at sentry.interfaces.stacktrace and maybe use that
 class FrameFilename:
     def __init__(self, frame_file_path: str) -> None:
-        # XXX: Using regexes would be better
-        if not frame_file_path or frame_file_path.find("<") > -1:
-            raise UnsupportedFrameFilename
+        # Using regexes would be better but this is easier to understand
+        if (
+            not frame_file_path
+            or frame_file_path[0] in ["[", "<", "/"]
+            or frame_file_path.find(" ") > -1
+            or frame_file_path.find("\\") > -1  # Windows support
+            or frame_file_path.find("/") == -1
```

Comment:
This can be the case when a file is at the top of source dir. We will look into supporting it in the future.